lib/ts/recipe/passwordless/utils.ts

```diff
@@ -158,3 +159,28 @@ export function defaultGuessInternationPhoneNumberFromInputPhoneNumber(
     }
     return value;
 }
+
+export async function getLoginAttemptInfoFromStorage(input: {
+    recipeImplementation: RecipeInterface;
+    userContext: any;
+}): Promise<
+    | {
+          deviceId: string;
+          preAuthSessionId: string;
+          flowType: PasswordlessFlowType;
+          contactInfo: string;
+          contactMethod: "EMAIL" | "PHONE";
+          lastResend: number;
+          redirectToPath?: string;
+      }
+    | undefined
+> {
+    return await input.recipeImplementation.getLoginAttemptInfo<{
+        contactInfo: string;
```

Comment:
I'd prefer if this was extracted into `types.ts` and reused everywhere we with `getLoginAttemptInfo`.

Reply:
The util function `getLoginAttemptInfoFromStorage` was introduced to solve this problem (I must have missed refactoring in this case). Ive changed the util function to use types declared in `types.ts` along with changing calls to `setLoginAttemptInfo` to use the types as well